.gitignore

```diff
@@ -1,2 +1,3 @@
 *.env.*
 .htaccess
+sites/*/profile_pictures/*
```

Comment:
I think this is OK.  @tmccormi @teryhill is the practice of including a gitignore in the repo agreed as proper?